openedx_caliper_tracking/tests/test_caliper_delivery.py

```diff
@@ -0,0 +1,86 @@
+"""
+This module contains the test cases for openedx_caliper_tracking
+application logs delivery to Rest API.
+"""
+import json
+import mock
+import os
+
+from django.conf import settings
+from django.test import TestCase, override_settings
+
+from openedx_caliper_tracking.processor import CaliperProcessor
+
+
+TEST_DIR_PATH = os.path.dirname(os.path.abspath(__file__))
```

Comment:
This line is repeated. Could we somehow avoid it?

Reply:
done in the